### DISCUSSION TITLE
SQL monitoring

### DISCUSSION THREAD

**Discussion by SooryaMuthuraj** [apache/skywalking#12011] [Q&A] [answered]
In SQL monitoring, Can we monitor  query ?

**Comment by wu-sheng** [comment]
What are you talking? Which part do you mean?

**Reply by SooryaMuthuraj** [reply]
In SQL monitoring, we are not able to monitor the slow queries.

**Reply by wu-sheng** [reply]
Be specific. Do you mean virtual database or MySQL(or other) server monitoring.

**Reply by SooryaMuthuraj** [reply]
In MYSQL

**Reply by wu-sheng** [reply, marked as answer]
That is not about query, that is MySQL it own slow SQL monitoring. 

Please refer to doc https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-mysql-monitoring/#collect-sampled-slow-sqls. Especially fluentbit setup.
